### PR TITLE
CEDS-3266 Play upgrade to version 2.8

### PR DESCRIPTION
**Pull request by MaciejRewera**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -4,31 +4,32 @@ import sbt._
...
 
-  private val wireMockVersion = "2.27.2"
   private val testScope = "test,it"
 
   val compile = Seq(
-    "uk.gov.hmrc"                     %% "simple-reactivemongo"       % "7.31.0-play-27",
+    "uk.gov.hmrc"                       %% "simple-reactivemongo"           % "8.0.0-play-28",
+    "uk.gov.hmrc"                       %% "bootstrap-backend-play-28"      % "5.3.0",
+    "uk.gov.hmrc"                       %% "wco-dec"                        % "0.35.0" excludeAll ExclusionRule("com.fasterxml.jackson.dataformat"),
     ws,
```
Comment (udinobi) [inline project/AppDependencies.scala:13]: Nit: is `ws` really required? (No harm to have it, I know)

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -4,31 +4,32 @@ import sbt._
...
-    "com.vladsch.flexmark"    %  "flexmark-all"       % "0.36.8"            % "test, it",
-    "org.pegdown"             %  "pegdown"            % "1.6.0"             % "test",
-    "com.typesafe.play"       %% "play-test"          % PlayVersion.current % "test",
-    "com.github.tomakehurst"  %  "wiremock"           % wireMockVersion     % testScope,
-    "org.mockito"             %  "mockito-core"       % "3.5.7"             % "test"
+    "org.scalatest"           %% "scalatest"          % "3.2.9"             % testScope,
+    "org.scalatestplus.play"  %% "scalatestplus-play" % "5.1.0"             % testScope,
+    "com.typesafe.play"       %% "play-test"          % PlayVersion.current % testScope,
+    "org.scalatestplus"       %% "mockito-3-4"        % "3.2.9.0"           % testScope,
+    "org.mockito"             %  "mockito-core"       % "3.5.7"             % testScope,
```
Comment (udinobi) [inline project/AppDependencies.scala:30]: Is `mockito-core` really required?

File: project/AppDependencies.scala (hunk 1 above)
Comment (MaciejRewera) [reply project/AppDependencies.scala]: Removed

File: project/AppDependencies.scala (hunk 2 above)
Comment (MaciejRewera) [reply project/AppDependencies.scala]: Removed

**Review by udinobi — APPROVED** [review]

**Pull request merged**